configure

```diff
@@ -230,8 +230,10 @@ fi
 require 'python3-mako' "You need the mako module for python3: see doc/INSTALL.md" python3 -c 'import mako'
 
 POSTGRES_INCLUDE=""
-if command -v pg_config 2> /dev/null; then
-    POSTGRES_INCLUDE="-I$(pg_config --includedir)"
+POSTGRES_LDLIBS=""
+if command -v "${PG_CONFIG-pg_config}" >/dev/null; then
+    POSTGRES_INCLUDE="-I$("${PG_CONFIG-pg_config}" --includedir)"
+    POSTGRES_LDLIBS="-L$("${PG_CONFIG-pg_config}" --libdir) -lpq"
```

Comment:
We could set the `$PG_CONFIG` at the top of the script to avoid having to use the get-or-default construct here.

Reply:
I've already left the computer for the night (going to bed), but I'll make this change within the day.

Reply:
[Implemented.](https://github.com/ElementsProject/lightning/pull/3995/commits/a29632dfbf4353b4868489a9a5e324e3da89a8e6)